Automattic/jetpack
# VideoPress Block: edit video settings while uploading

Allows users to edit some video settings while the video is uploading. Users can set the poster image and change the title.




#### Changes proposed in this Pull Request:

* Adds a new screen on video upload for editing video settings.
* Upload Poster tab
* Select poster from frame tab.
* Edit title text field.

#### Jetpack product discussion


pxWta-1pr-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:

* D81302-code to be applied too if it is not already merged. In that case, public api needs to be sandboxed.
* Create a new post and upload videos. You see an expandable element for editing video settings.
<img width="617" alt="Screenshot 2022-05-31 at 5 25 31 PM" src="https://user-images.githubusercontent.com/500744/171197675-049736ab-d733-40c5-ab65-f607f2924002.png">

* Expand, and try different combinations of  changing the title, uploading a poster, or selecting a frame as a poster on each video.
* Pressing "Done" should bring up the video embed. Verify that the settings you updated took effect.
* Uploading a video without expanding the editor should work the same way it did.

<img width="632" alt="Screenshot 2022-05-31 at 5 22 14 PM" src="https://user-images.githubusercontent.com/500744/171197305-75b7e997-1e49-4f1a-bf2b-4c950701e1d3.png">
<img width="640" alt="Screenshot 2022-05-31 at 5 21 56 PM" src="https://user-images.githubusercontent.com/500744/171197323-9c4414aa-f9d5-48f8-ba4d-7201ffe1433d.png">
<img width="620" alt="Screenshot 2022-05-31 at 5 20 37 PM" src="https://user-images.githubusercontent.com/500744/171197342-8e602abb-66b6-4762-bacf-0774b21a79a9.png">
<img width="617" alt="Screenshot 2022-05-31 at 5 20 19 PM" src="https://user-images.githubusercontent.com/500744/171197348-3f74ed41-f8da-4c54-91d6-aff948d5787a.png">
<img width="627" alt="Screenshot 2022-05-31 at 5 20 08 PM" src="https://user-images.githubusercontent.com/500744/171197354-99095aff-355f-41fe-8d4e-7b49c6598d5e.png">
